db/migrate/20230511160143_create_targets.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTargets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :targets do |t|
+      t.references :topic, null: false, foreign_key: true
+      t.string :tittle
+      t.float :radius
+      t.float :lat
+      t.float :lng
```

Comment:
remember to add database validation too here when checking for `presence: true` on the model:
`null: false`

Reply:
you right! thanks